src/detectEyes.py

```diff
@@ -1,27 +1,141 @@
 import cv2
+from enum import Enum
 
-def detectEyes(image, detector):
-    '''Returns the coordinates of the eyes in the image.'''
+def centerCoordinates(f):
+    '''Decorator function to convert corner coordinates to center coordinates'''
+    def inner(*args, **kwargs):
+        eyes = f(*args, **kwargs)
+        return [map(round, (x+w/2, y+h/2)) for (x, y, w, h) in sorted(eyes, key=lambda eye: eye[1])[:2]]
+    return inner
+
+class DetectionType(Enum):
+    EYE_CASCADE = 1
+    EYE_CASCADE_BLOB = 2
+    FACE_EYE_CASCADE = 3
+    FACE_EYE_CASCADE_BLOB = 4
+    EIGENFACE = 4
+
+def detectEyes(image, detectorType=DetectionType.EYE_CASCADE, eyeDetector=None, blobDetector=None, faceDetector=None):
+    '''Returns the coordinates of the eyes in the image using the specified detector'''
+    if detectorType == DetectionType.EYE_CASCADE:
+        return eyeCascadeDetector(image, eyeDetector)
+
+    elif detectorType == DetectionType.EYE_CASCADE_BLOB:
+        return eyeCascadeBlobDetector(image, eyeDetector, blobDetector)
+
+    elif detectorType == DetectionType.FACE_EYE_CASCADE:
+        return faceEyeCascadeDetector(image, eyeDetector, faceDetector)
+
+    elif detectorType == DetectionType.FACE_EYE_CASCADE_BLOB:
+        return faceEyeCascadeBlobDetector(image, detector)
```

Comment:
`pylint` throws a warning for this line - too many positional arguments.

Reply:
All of those functions are just placeholders for now, I'll implement them later.